src/Compilers/Core/CodeAnalysisTest/Text/StringTextTest.cs

```diff
@@ -72,7 +72,7 @@ public void FromStream_Errors()
         {
             Assert.Throws<ArgumentNullException>(() => SourceText.From((Stream)null, Encoding.UTF8));
             Assert.Throws<ArgumentException>(() => SourceText.From(new TestStream(canRead: false, canSeek: true), Encoding.UTF8));
-            Assert.Throws<ArgumentException>(() => SourceText.From(new TestStream(canRead: true, canSeek: false), Encoding.UTF8));
+            Assert.Throws<NotImplementedException>(() => SourceText.From(new TestStream(canRead: true, canSeek: false), Encoding.UTF8));
```

Comment:
>NotImplementedException [](start = 26, length = 23)

Where are we throwing `NotImplementedException`? If it's from `TestStream`, does that class need to be updated?

Reply:
This is no backing stream in this case, so TestStream throws from: https://github.com/dotnet/roslyn/blob/c250d31757c0815c45f344d60c27e50a5e50960e/src/Test/Utilities/Portable/Mocks/TestStream.cs#L102-L109

which seems correct.